src/Language/DifferentialDatalog/Attribute.hs

```diff
@@ -140,6 +141,20 @@ fieldValidateAttr d attr = do
          "deserialize_from_array" -> return ()
          n -> err d (pos attr) $ "Unknown attribute " ++ n
 
+relValidateAttr :: (MonadError String me) => DatalogProgram -> Attribute -> me ()
+relValidateAttr d attr = do
+    case name attr of
+         "original" -> case attrVal attr of
+              E (EString _ _) -> return ()
+              _ -> err d (pos attr) $ "The value of the 'original' attribute should be a string, e.g., #[original=\"Foo\"]"
```

Comment:
```suggestion
              _ -> err d (pos attr) $ "The value of the 'original' attribute should be a string, e.g. #[original = \"Foo\"]"
```

Reply:
According to this https://en.wiktionary.org/wiki/e.g. you should use a comma after e.g. in US English.